formatting-code/asparuh/JavaScript-code.js

```diff
@@ -0,0 +1,121 @@
+b=navigator.appName;
+
+addScroll=false;

```

Comment:
After line #8 declared variables have properly var statement, I think i should have the same for 'b' and 'addScroll'?